tests/go/lang/logger_leak/testdata/main.go

```diff
@@ -0,0 +1,72 @@
+package go_lang_logger_leak
+
+import (
+	seelog "github.com/cihub/seelog"
+	glog "github.com/golang/glog"
+	zerolog "github.com/rs/zerolog"
+	logrus "github.com/sirupsen/logrus"
+	"go.uber.org/zap"
+)
+
+func bad(Data any) {
+	// bearer:expected go_lang_logger_leak
+	zerolog.Error().Msg(Data)
+	// bearer:expected go_lang_logger_leak
+	zerolog.Error().Msgf("Some data: %#v", Data)
```

Comment:
These are also counted as string literals, as in the Java case

Reply:
https://github.com/Bearer/bearer-rules/actions/runs/8001834206/job/21853837960?pr=301